src/main/ml-modules/ext/serviceModel.sjs

```diff
@@ -153,7 +154,7 @@ function getLayerModelIndexEntryFromDb(serviceModelEntry, layerId) {
         }
         */
     let layerMetadataField =
-        fn.head(xdmp.invokeFunction(function () {
+        fn.head(ampedFunctions.invokeFunction(function () {
```

Comment:
I think this is the right pattern in general - make a library that exists solely for wrapping ML functions with amps, so that all amping is done in one place. Makes things easy to understand and manage.